### ISSUE TITLE
Clear unwanted warnings in Refine

### ISSUE THREAD

**Issue by kristenwilson** [k-int/gokb-phase1#208]
Ability to clear or push to the bottom Refine warnings you don't intend to fix.

**Comment by iyawnis** [comment]
Suggest we push this back to 5.0, as I need more time to start working on refine.

**Comment by kristenwilson** [comment]
I'm fine with that. I will change the milestone.

**Comment by iyawnis** [comment]
How do you expect the clearing to persist? The simplest solution is simply adding close buttons on validation messages, but then those will re-appear on next validation. I guess this is not useful to you?

**Comment by kristenwilson** [comment]
If clearing the messages temporarily is an easy solution, I think that would still be useful. A lot of times people work through the entire validation process in one sitting, and it's nice to be be able to get rid of the things you know you are not worrying about. It would be ideal to have them be permanently dismissed if there's a way to make that work.

**Comment by sosguthorpe** [comment]
I think there is a misunderstanding on exactly when the message will reappear. The message would reappear after the very next edit of anything in the project. This means a user could close a warning, fix the next warning by editing a single value in a cell and consequently the message previously closed to reappear again. The close would not be observed for a sitting, but only until next validation.

Ioannis is currently working on getting these message closes to be observed for the refine session. We will then extend on top of that to make the settings persist across the refine sessions. Meaning settings will be remembered locally per project. This functionality is still not planned for release until 5.0.

**Comment by iyawnis** [comment]
@kristenwilson I have implemented part of this functionality. Hidden messages will remain hidden as long as refine is running. They will persist login/logout, and are specific to project and user. The hidden messages will re-appear when refine is restarted. Making them persist between restarts is possible although will take a lot of development time, so I suggest we leave that part for the future. Closing the attached warning message, will make hidden messages visible. This will always be at the bottom of the list. 
![screenshot from 2015-03-23 14 46 17](https://cloud.githubusercontent.com/assets/2336352/6782445/2fa87a70-d16c-11e4-9dcc-a367480a44c8.png)
